digital_image_processing/image_watersheding_custom_markers.py

```diff
@@ -0,0 +1,89 @@
+import cv2
```

Comment:
Please add a URL to a site that discusses what watersheding and custom markers are and where/when/why we  would want to use this algrithm.

Reply:
https://www.pyimagesearch.com/2015/11/02/watershed-opencv/

Here it is. Will include this.